cmake/find_grpc.cmake

```diff
@@ -18,3 +18,11 @@ message(STATUS "Using ZLIB: ${ZLIB_INCLUDE_DIRS}, ${ZLIB_LIBRARIES}")
 
 find_package(gRPC CONFIG REQUIRED)
 message(STATUS "Using gRPC: ${gRPC_VERSION}")
+
+set(_PROTOBUF_LIBPROTOBUF protobuf::libprotobuf)
+set(_PROTOBUF_PROTOC $<TARGET_FILE:protobuf::protoc>)
+set(_GRPC_GRPCPP_UNSECURE gRPC::grpc++_unsecure)
+set(_GRPC_GRPCPP gRPC::grpc++)
+set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:gRPC::grpc_cpp_plugin>)
+
+set(gRPC_FOUND TRUE)
```

Comment:
If find_package(gRPC CONFIG `REQUIRED`) success, gRPC is already found.

Reply:
Yes, this is mainly to keep the structure consistent with TiCS.